projects/components/src/table/cells/csv-generators/table-cell-boolean-csv-generator.ts

```diff
@@ -0,0 +1,13 @@
+import { TableCellCsvGeneratorBase } from '../table-cell-csv-generator-base';
+import { Injectable } from '@angular/core';
+import { isUndefined } from 'lodash-es';
+
+@Injectable({ providedIn: 'root' })
+export class TableCellBooleanCsvGenerator implements TableCellCsvGeneratorBase<boolean> {
+  public readonly type = 'CSV_GENERATOR';
+  public readonly cellType: string = 'boolean';
```

Comment:
Does this need to match `CoreTableCellCsvGeneratorType.Boolean` ? if so, can we use that value

Reply:
We are getting rid of csv generator type and directly linking them to cell renderer types.